lib/util/save-async-stack.js

```diff
@@ -2,8 +2,13 @@ module.exports = function saveAsyncStack(instance, lines) {
   if (instance.client.config.asyncStackTraces) {
     // a hack to get a callstack into the client code despite this
     // node.js bug https://github.com/nodejs/node/issues/11865
-    const stackByLines = new Error().stack.split('\n');
-    stackByLines.splice(0, lines);
-    instance._asyncStack = stackByLines;
+
+    // Save error here but not error trace
+    // reading trace with '--enable-source-maps' flag on node can be very costly
+
+    instance._asyncStack = {
```

Comment:
This is a breaking change, right?

Reply:
`_` means private var ;-) and not exposed in any docs, so it depends